source/transactions/tests/unified/mongos-unpin.json

```diff
@@ -142,7 +160,7 @@
       ]
     },
     {
-      "description": "unpin after TransientTransactionError error on abort",
+      "description": "unpin after non-transient error on abort",
```

Comment:
I assume these were mis-labeled?

Reply:
Yep. This test uses the non-transient code 24 error.